### ISSUE TITLE
Incorrect error: `<colgroup>` without `span` attribute and empty content

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#398]
The following is valid HTML, but markuplint gives a `permitted-contents` error.

```html
<table>
  <colgroup></colgroup> <!-- ← error -->
  <colgroup><col /></colgroup> <!-- ← no errors -->
  <colgroup span="1"></colgroup> <!-- ← no errors -->
  <tbody>
    <tr>
      <td></td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>
```

[`<colgroup>` element spec](https://html.spec.whatwg.org/multipage/tables.html#the-colgroup-element):

> Content model:
> If the span attribute is absent: **Zero** or more col and template elements.
